Justfile

```diff
@@ -1,19 +1,34 @@
+src_dir := "src"
 tests_dir := "tests"
-code_dirs := "src" + " " + tests_dir
-run_prefix := if env_var_or_default("VIRTUAL_ENV", "") == "" { "uv run " } else { "" }
+code_dirs := src_dir + " " + tests_dir
 
+# list available rules
+default:
+    just --list
+
+# build the project
+build:
+	uv build
+
+# install dependencies
+sync:
+	uv sync --group lint --group test --group tox
+
+# check the code
 check:
-	{{ run_prefix }}ruff format --check {{ code_dirs }}
```

Comment:
Why remove the run_prefix stuff? Without it, it's not possible to use just without activating the virtualenv first, right?

In cattrs, I went a slightly different way and removed tox altogether in favor of uv and just. Maybe we could do the same here

Reply:
Yeah, without the prefix activating the venv is important otherwise commands/tools/etc may not be resolved. Still, it is possible to always use `uv run` approach and uv should handle the things. So, I can get this prefix back or prefixate `uv run` to every command, just tell what approach would you prefer. In this PR I would like to avoid infrastructural chores concerning just, uv, tox and CI/CD (if necessary), I think the changes of that sort are good to be addressed in another PR + this PR is a final pre-step towards the #193 which I would like to focus on.